docs/100_producten/100_borgstelling/050_uitbetalen-borgstelling/index.md

```diff
@@ -22,7 +22,7 @@ Start het proces uitbetalen borgstelling.
 
 ### Betaal borgstelling uit
```

Comment:
@nielserkamp-ritense is deze nog nodig of kan de logica worden verplaatst naar het end event borgstelling uitbetaald dat daar de gebeurtenis aan een wachtrij wordt toegevoegd?

Reply:
Ik zou deze vanwege duidelijkheid van het process er in laten staan aangezien het een daadwerkelijk stap in het process is en een andere taak is dan het end event.